packages/core/src/constants/chainId.ts

```diff
@@ -1,3 +1,40 @@
+import { Localhost, Hardhat } from '../model/chain/local'
+import { Avalanche } from '../model/chain/avalanche'
+import { BSC, BSCTestnet } from '../model/chain/bsc'
+import { Fantom } from '../model/chain/fantom'
+import { Harmony } from '../model/chain/harmony'
+import { Mainnet, Ropsten, Rinkeby, Goerli, Kovan } from '../model/chain/ethereum'
+import { Moonriver, MoonbaseAlpha } from '../model/chain/moonriver'
+import { Palm } from '../model/chain/palm'
+import { Polygon, Mumbai } from '../model/chain/polygon'
+import { Songbird } from '../model/chain/songbird'
+import { Theta, ThetaTestnet } from '../model/chain/theta'
+import { xDai } from '../model/chain/xdai'
+
+export const DEFAULT_SUPPORTED_CHAINS = [
+  Localhost,
+  Hardhat,
+  Avalanche,
+  Mainnet,
+  Ropsten,
+  Rinkeby,
+  Goerli,
+  Kovan,
+  BSC,
```

Comment:
rough alphabet order, put network from the same chain together

Reply:
👍